packages/primitives/src/icons/contact-support-16.svg

```diff
@@ -0,0 +1,31 @@
+<!--
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Our naming convention doesn't address the scenario but rather explains the image as it is..so shall we rename the icon name of `ContactSupportIcon` ?

Eg - Logout button is named as `arrow-right-from-bracket-16`

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/245/commits/f359bc0db19f27b4311507704d31856cfd4fa32b